### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by AswathyVNair**

### CODE REVIEW

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    width: 90%;
+    height:20rem;
+    justify-content: space-between;
+}
+    
+.alert-item{
+    height: 3.5rem;
+    display: flex;
+    align-items: center;
+    border-radius: 0.3rem;
```
Comment (Omkar-Ghate) [inline alert/alert.css]: can use px here

**Review by Omkar-Ghate — APPROVED** [review]
Looks good for me.

**Review by DivyanshSareen — APPROVED** [review]
good to merge

File: alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    
+.danger-alert{    
+    color: var(--red);    
+    background-color:var(--light-red);
+    border-left:5px solid var(--red)
+}
+    
+i,.alert-message{
+    padding-left: 1rem;
+
```
Comment (rutu14) [inline alert/alert.css]: Extra spacing can be discarded.

**Review by rutu14 — APPROVED** [review]

File: alert/alert.css (hunk 1 above)
Comment (AswathyVNair) [reply alert/alert.css]: Ok Omkar, I have made the changes

File: alert/alert.css (hunk 2 above)
Comment (AswathyVNair) [reply alert/alert.css]: Yes, I have removed.

**Pull request merged**